auth/idChecker.js

```diff
@@ -0,0 +1,11 @@
+function idChecker(req, res, next) {
+    const userID = req.get('UserID');
```

Comment:
I'm not sure how useful this is as you already have the user ID in the token payload. 🤔 This is good code though.